src/Entity/OfferingQuota.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace SandwaveIo\Acronis\Entity;
+
+use JMS\Serializer\Annotation as Serializer;
+
+class OfferingQuota
+{
+    /**
+     * @var int
+     * @Serializer\SerializedName("version")
+     * @Serializer\Type("integer")
+     * @Serializer\Groups({"update_data"})
+     */
+    private $version;
+
+    /**
+     * @var int|null
+     * @Serializer\SerializedName("overage")
+     * @Serializer\Type("integer")
+     * @Serializer\Groups({"update_data"})
+     */
+    private $overage;
+
+    /**
+     * @var int|null
+     * @Serializer\SerializedName("value")
+     * @Serializer\Type("integer")
+     * @Serializer\Groups({"update_data"})
+     */
+    private $value;
+
+    public function __construct(int $version, ?int $overage, ?int $value)
```

Comment:
parameters zijn nullable, maar moet je altijd meegeven
is dit een bewuste keuze?

als je nieuw object wil waarbij overage en value null is, dan moet je die zo dus aanroepen
`new OfferingQuota(1, null, null)`
volgende gooit een **ArgumentCountError**
`new OfferingQuota(1)`